Add pangeo launch blog to website (#85)
2i2c-org/2i2c-brochure: content/posts/2021/pangeo-goes-live/index.md

---
title: "Pangeo goes live on 2i2c!"
subtitle: ""
summary: "2i2c are pleased to announce that the first Pangeo JupyterHub is now live on 2i2c-operated infrastructure! :tada: "
authors: ["Sarah Gibson"]
tags: [2i2c, pangeo]
categories: [updates]
date: 2021-11-08
featured: false
draft: false
---

We have recently finished re-creating the Pangeo community JupyterHub hosted on GCP in the [2i2c-org/infrastructure](https://github.com/2i2c-org/infrastructure) repository.
This is a huge milestone in our partnership with Pangeo to provide expertise and operations of cloud-based, vendor-agnostic Jupyter infrastructure and workflows.

For users of Pangeo's infrastructure, the switch should be a smooth one!
The new hub should behave nearly identically to the old one, and will be managed by 2i2c engineers moving forward, in partnership with the Pangeo community.

> It will be available at the same URL (https://us-central1-b.gcp.pangeo.io) and your home directories were synced to the new hub only a few days before the migration took place.

Author: I switch tenses halfway through this sentence (from future to past) 😕

Author: ```suggestion
It will be available at the same URL (https://us-central1-b.gcp.pangeo.io) and there's no need to worry about your home directories, they were synced to the new hub only a few days before the migration took place.
```

Should resolve the tenses issue

> Development and operations on this hub will all be done in the open, and [we invite participation and feedback from others]() in our infrastructure work.

Author: What's an appropriate link here?

Reviewer: Porting my comment from the hackmd 😉 

> Mmm... if we are requesting feedback maybe we could create some specific form? Or maybe some discourse post for getting feedback?

Author: Hmm, maybe this one? https://discourse.pangeo.io/t/migration-of-us-central1-b-gcp-pangeo-io-to-2i2c-infrastructure/1890

Author: ```suggestion
Development and operations on this hub will all be done in the open, and [we invite participation and feedback from others](https://discourse.pangeo.io/t/migration-of-us-central1-b-gcp-pangeo-io-to-2i2c-infrastructure/1890) in our infrastructure work.
```

Author: I am unresolving this after a Slack discussion yesterday

> we should also be encouraging the Pangeo community to open support tickets as well 

**cc:** @choldgraf

Reviewer 2: I opened up an issue to discuss this here: https://github.com/2i2c-org/team-compass/issues/297 

I think our goal should be to develop a prototype for what this kind of support might be, and try it out with the the Pangeo community to see if it feels sustainable. Does that make sense?

Modern scalable cloud infrastructure is one example of this. Maintaining a complex JupyterHub with many users is a difficult task, and has required significant resources from the Pangeo Project up to this point.

## The Pangeo-2i2c Partnership

[2i2c](https://2i2c.org) is a non-profit team that develops and operates cloud infrastructure for interactive computing workflows.
We have extensive experience in Jupyter workflows in the cloud and a long history of contributions to projects in this ecosystem.
We have built a cloud deployment management system that allows us to centralise and configure the deployment of many independent JupyterHubs, empowering communities to leverage the same infrastructure (and team!) for JupyterHubs running in the cloud.

Similarly to Pangeo, all of 2i2c's core infrastructure is cloud- and vendor-agnostic, and follows a model of building open source tools and giving back to those communities.
Our partnership with Pangeo began through 2i2c's core competency in these areas and the similarity between the two project's technical stacks.

> We've begun a partnership wherby 2i2c will manage Pangeo's cloud infrastructure and lead efforts to develop new features, in partnership with open source communities.

Reviewer: wherby?

Author: Should be whereby

Author: ```suggestion
We've begun a partnership whereby 2i2c will manage Pangeo's cloud infrastructure and lead efforts to develop new features, in partnership with open source communities.
```

> We've begun a partnership whereby 2i2c will manage Pangeo's cloud infrastructure and lead efforts to develop new features, in partnership with open source communities.

Author: ```suggestion
We've begun a partnership whereby 2i2c will manage Pangeo's cloud infrastructure and lead efforts to develop new features, in partnership with open source communities.
We sketched out a few ideas to focus on in this [kick-off thread on Discourse](https://discourse.pangeo.io/t/notes-from-the-pangeo-2i2c-kick-off-meeting/1587).
```

This approach allows each community to focus on it's core strengths: Pangeo will continue to grow an open community and scientific software ecosystem around geospatial analytics, and 2i2c will oversee the development and operations of the core cloud infrastructure stack that powers Pangeo's workflows.

> We are extremely grateful to the Pangeo project for giving us the opportunity to serve their community, and we look forward to a long partnership ahead! :rocket:

Author: From the HackMD:

> Would be nice to have a final section hinting at some of the plans we have for the future.

I'm not sure we've decided this yet? I guess the next step is to migrate the GCP Binder? Is that the kind of work that we should hint at here?

Reviewer 2: I think that in this case we can just leave it relatively hand-wavy and point to that Discourse post to give folks an idea of what might be done

Author: I added that discourse post a little further up the previous paragraph for flow.

---
title: "Pangeo goes live on 2i2c!"
subtitle: ""
summary: "2i2c are pleased to announce that the first Pangeo JupyterHub is now live on 2i2c-operated infrastructure! :tada: "
authors: ["Sarah Gibson"]
tags: [2i2c, pangeo]
categories: [updates]

> date: 2021-11-08

Author: Add new date before merging!!!

featured: false
draft: false
---

We have recently finished re-creating the Pangeo community JupyterHub hosted on GCP in the [2i2c-org/infrastructure](https://github.com/2i2c-org/infrastructure) repository.
This is a huge milestone in our partnership with Pangeo to provide expertise and operations of cloud-based, vendor-agnostic Jupyter infrastructure and workflows.

> For users of Pangeo's infrastructure, the switch should be a smooth one!

Author: ```suggestion
For users of Pangeo's infrastructure, the switch should have been a smooth one!
```

Similarly to Pangeo, all of 2i2c's core infrastructure is cloud- and vendor-agnostic, and follows a model of building open source tools and giving back to those communities.
Our partnership with Pangeo began through 2i2c's core competency in these areas and the similarity between the two project's technical stacks.

We've begun a partnership whereby 2i2c will manage Pangeo's cloud infrastructure and lead efforts to develop new features, in partnership with open source communities.
We sketched out a few ideas to focus on in this [kick-off thread on Discourse](https://discourse.pangeo.io/t/notes-from-the-pangeo-2i2c-kick-off-meeting/1587).
This approach allows each community to focus on it's core strengths: Pangeo will continue to grow an open community and scientific software ecosystem around geospatial analytics, and 2i2c will oversee the development and operations of the core cloud infrastructure stack that powers Pangeo's workflows.

> 
> We are extremely grateful to the Pangeo project for giving us the opportunity to serve their community, and we look forward to a long partnership ahead! :rocket:

Reviewer 2: ```suggestion
In some areas we are still experiment with different collaboration models to ensure that the needs of the Pangeo community are met in a way that is also sustainable for 2i2c. Over the coming weeks, you may see some conversations (and threads for feedback!) about different support and operations models that work best for the community. We are excited to use this as an opportunity to learn more about how to serve more complex and diverse communities like Pangeo.

We are extremely grateful to the Pangeo project for giving us the opportunity to serve their community, and we look forward to a long partnership ahead! :rocket:
```

Really good point about languishing blog posts - how about this language, and a plan like the following:

- Post this blog post on Monday the 15th
- As a support stop-gap, @sgibson91 keep an eye on the Pangeo Discourse for issues that seem related to 2i2c SRE.
- Over the next few weeks we work out a model for balancing `support@2i2c.org` and @sgibson91's role regarding operations for the Pangeo hubs (or, whoever else is primarily working with Pangeo if we bring another person to supplement this role when she moves to partial EOSS funding)

Does that work?

Author: Yep, sounds good to me! 🤞🏻 there aren't many more issues - I think the netpol stuff is at least put to bed.

categories: [updates]
date: 2021-11-08
featured: false
draft: false
---

We have recently finished re-creating the Pangeo community JupyterHub hosted on GCP in the [2i2c-org/infrastructure](https://github.com/2i2c-org/infrastructure) repository.
This is a huge milestone in our partnership with Pangeo to provide expertise and operations of cloud-based, vendor-agnostic Jupyter infrastructure and workflows.

For users of Pangeo's infrastructure, the switch should have been a smooth one!
The new hub should behave nearly identically to the old one, and will be managed by 2i2c engineers moving forward, in partnership with the Pangeo community.
It will be available at the same URL (https://us-central1-b.gcp.pangeo.io) and there's no need to worry about your home directories, they were synced to the new hub only a few days before the migration took place.
Development and operations on this hub will all be done in the open and we invite participation and feedback from others in our infrastructure work.
Please see [this Discourse thread](https://discourse.pangeo.io/t/migration-of-us-central1-b-gcp-pangeo-io-to-2i2c-infrastructure/1890) as an initial place to provide feedback.

> On **15th November 2021**, the old Pangeo GCP JupyterHub will be shut down, and the project will move forward on the new 2i2c Pangeo Hub.

Author: ```suggestion
On **22nd November 2021**, the old Pangeo GCP JupyterHub will be shut down, and the project will move forward on the new 2i2c Pangeo Hub.
```

Suggesting a new shutdown date of the old cluster

---

> title: "Pangeo goes live on 2i2c!"

Reviewer 3: ```suggestion
title: "Pangeo Cloud goes live on 2i2c!"
```

subtitle: ""
summary: "2i2c are pleased to announce that the first Pangeo JupyterHub is now live on 2i2c-operated infrastructure! :tada: "
authors: ["Sarah Gibson"]
tags: [2i2c, pangeo]
categories: [updates]
date: 2021-11-08
featured: false
image:
  preview_only: true
draft: false
---

> 

Reviewer 3: ```suggestion

 [Pangeo Cloud](https://pangeo.io/cloud.html) is an experimental service providing public cloud-based data-science environments for data-intensive geoscience research.
```

> We have recently finished re-creating the Pangeo community JupyterHub hosted on GCP in the [2i2c-org/infrastructure](https://github.com/2i2c-org/infrastructure) repository.

Reviewer 3: ```suggestion
We have recently finished re-creating the [Pangeo](http://pangeo.io/) community JupyterHub hosted on GCP in the [2i2c-org/infrastructure](https://github.com/2i2c-org/infrastructure) repository.
```

This is a huge milestone in our partnership with Pangeo to provide expertise and operations of cloud-based, vendor-agnostic Jupyter infrastructure and workflows.

> For users of Pangeo's infrastructure, the switch should have been a smooth one!

Reviewer 3: ```suggestion
For users of Pangeo Cloud, the switch should have been a smooth one!
```

---
title: "Pangeo Cloud goes live on 2i2c!"
subtitle: ""
summary: "2i2c are pleased to announce that the first Pangeo JupyterHub is now live on 2i2c-operated infrastructure! :tada: "
authors: ["Sarah Gibson"]
tags: [2i2c, pangeo]
categories: [updates]

> date: 2021-11-08

Reviewer 2: ```suggestion
date: 2021-11-16
```

We have recently finished re-creating the [Pangeo](http://pangeo.io/) community JupyterHub hosted on GCP in the [2i2c-org/infrastructure](https://github.com/2i2c-org/infrastructure) repository.
This is a huge milestone in our partnership with Pangeo to provide expertise and operations of cloud-based, vendor-agnostic Jupyter infrastructure and workflows.

For users of Pangeo Cloud, the switch should have been a smooth one!
The new hub should behave nearly identically to the old one, and will be managed by 2i2c engineers moving forward, in partnership with the Pangeo community.
It will be available at the same URL (https://us-central1-b.gcp.pangeo.io) and there's no need to worry about your home directories, they were synced to the new hub only a few days before the migration took place.
Development and operations on this hub will all be done in the open and we invite participation and feedback from others in our infrastructure work.
Please see [this Discourse thread](https://discourse.pangeo.io/t/migration-of-us-central1-b-gcp-pangeo-io-to-2i2c-infrastructure/1890) as an initial place to provide feedback.

> On **15th November 2021**, the old Pangeo GCP JupyterHub will be shut down, and the project will move forward on the new 2i2c Pangeo Hub.

Reviewer 2: ```suggestion
On **22nd November 2021**, the old Pangeo GCP JupyterHub will be shut down, and the project will move forward on the new 2i2c Pangeo Hub.
```
